build-files/vadr/1.6.4-rsv/Dockerfile

```diff
@@ -79,11 +84,14 @@ CMD ["v-annotate.pl", "-h"]
 FROM app AS test 
 
 # testing that executable is in PATH
-RUN v-annotate.pl -h
-RUN fasta-trim-terminal-ambigs.pl -h || echo "prints to stderr for some reason"
+# v-build.pl -h to ensure required perl modules are available (See issue #1393)
+RUN v-annotate.pl -h &&  \
+v-build.pl -h && \
+fasta-trim-terminal-ambigs.pl -h || echo "prints to stderr for some reason"
 
 # running vadr tests
-RUN /opt/vadr/vadr/testfiles/do-install-tests-local.sh
+#NOTE: These tests require the flavi and calici models to be present, which are not included in this image. These tests are however run in the docker build process for the `build-files/vadr/1.6.4-2505/Dockerfile` image.
```

Comment:
This basically removes the test for the rsv image. Are we okay with that?

Reply:
rsv and all of the other images too, except the dockerfile for `1.6.4-2505`

I'm fine with it but another option would be to download the flavi & calici models in the test stage (so they aren't in the final image/app stage) and run the test script. It adds quite a bit of time to the build process & it feels a little duplicative to test across all the images

Reply:
The other images have virus-specific tests (i.e. the sars-cov-2 one downloads a sars-cov-2 sequence and runs it against the model). The rsv image doesn't test to ensure the included model works, so the test stage for 1.6.4-rsv doesn't actually do anything.

Reply:
Ah, thank you for clarifying. 

I can run the RSV-specific test on  a computer with enough RAM to verify real quick. I'll copy/paste the output when it's finished